app/views/staff/training_sessions/show.html.erb

```diff
@@ -127,4 +133,10 @@
     <%= submit_tag "Update", class: 'save-button' %>
   <% end %>
 
+  <% if @user.admin? %>
```

Comment:
In addition to showing and hiding the button to submit, you should add checks to the controller to make sure the current user is an `admin`.